zenip-42000.md

```diff
@@ -0,0 +1,558 @@
+- [References](#references)
+<!--te-->
+
+## Terminology
+
+The key words "MUST", "SHOULD", "SHOULD NOT", "MAY", "RECOMMENDED",
+"OPTIONAL", and "REQUIRED" in this document are to be interpreted as
+described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+## Abstract
+
+A Horizen Improvement Proposal (ZenIP) is a design document providing
+information to the Horizen community, or describing a new feature for
+Horizen or its processes or environment. The ZenIP should provide a
+concise technical specification of the feature and a rationale for the
+feature.
+
+We intend ZenIPs to be the primary mechanism for proposing new features,
+for collecting community input on an issue, and for documenting the
+design decisions that have gone into Horizen. The Owner(s) of the ZenIP
+(usually the authors(s)) are responsible for building consensus within
+the community and documenting dissenting opinions.
+
+Because the ZenIPs are maintained as text files in a versioned
+repository, their revision history is the historical record of the
+feature proposal.
+
+This document is based on the work done by @acityinohio, @daira, and @str4d
+with [zip-0000](https://github.com/zcash/zips/blob/master/zip-0000.rst).
+They themselves based their document partly on the work done by @luke-jr with [BIP2](https://github.com/bitcoin/bips/blob/master/bip-0002.mediawiki).
+
+## Motivation
+
+It has been the stated goal of Horizen from the very beginning to build a decentralized platform, where not just the infrastructure is distributed, but also the funding and decision making. Part of the Horizen block subsidy goes to the Zen Blockchain Foundation treasury (at the time of writing - block 571074 - 20%), from where it gets distributed to fund the development of the protocol, the sidechain implementation, marketing as well as business development.
+
+The ZenIP process will be an important step towards a decentralized decision making process for the Horizen ecosystem. Improvement Proposal processes have proven to work well for decentralized blockchain projects, hence it should be the best way to distribute influence and power towards the broader community of open source contributors and stakeholders.
+
+## Specification
+
+### ZenIP Workflow
+
+The ZenIP process begins with a new idea for Horizen. Each potential
+ZenIP must have an Owner - someone who writes the ZenIP using the style
+and format described below, shepherds the discussions in the appropriate
+forums, and attempts to build community consensus around the idea. The
+ZenIP Owner should first attempt to ascertain whether the idea is
+ZenIP-able. Small enhancements or patches to a particular piece of
+software often don't require standardisation between multiple projects;
+these don't need a ZenIP and should be injected into the relevant
+project-specific development workflow with a patch submission to the
+applicable issue tracker.
+
+Additionally, some ideas might have been brought forward for changing Horizen
```

Comment:
The meaning of this sentence is not too clear to me :)

Reply:
Yep I was referring to line 94, "Additionally, some ideas might have been brought forward for changing Horizen that have been rejected for various reasons." Ok, now I understand what you meant.  I still think this needs to be written in a more clear way, maybe along the line of you comment above: before submitting a new ZenIP, please check previous proposals, including rejected ones.

Reply:
please check if my last commit addresses your concern sufficiently @mauriziobinello

Reply:
yep it looks good to me